common/h/registers/x86_64_regs.h

```diff
@@ -308,6 +318,7 @@ namespace Dyninst { namespace x86_64 {
   DEF_REGISTER(     xmm29,   0x1D |  XMMS |   XMM | Arch_x86_64, "x86_64");
   DEF_REGISTER(     xmm30,   0x1E |  XMMS |   XMM | Arch_x86_64, "x86_64");
   DEF_REGISTER(     xmm31,   0x1F |  XMMS |   XMM | Arch_x86_64, "x86_64");
+  DEF_REGISTER(     mxcsr,   0x20 | D_REG |   XMM | Arch_x86_64, "x86_64");
```

Comment:
`XMM` category needs to be changed as above to a new category such as `FPCTL`.

Reply:
Same as above.